zerver/lib/actions.py

```diff
@@ -2576,17 +2587,18 @@ def gather_subscriptions_helper(user_profile):
             unsubscribed.append(stream_dict)
 
     never_subscribed_stream_ids = [item for item in all_streams_id if item not in sub_unsub_stream_ids]
-    for stream_id in never_subscribed_stream_ids:
-        stream = Stream.objects.get(id=stream_id)
-        subscribers = get_subscriber_ids(stream)
-        stream_dict = {'name': stream.name,
-                       'invite_only': stream.invite_only,
-                       'stream_id': stream.id,
-                       'description': stream.description,
-                       'email_address': encode_email_address_helper(stream.name, stream.email_token)}
-        if subscribers is not None:
-            stream_dict['subscribers'] = subscribers
-        never_subscribed.append(stream_dict)
+    never_subscribed_streams = [streamx for streamx in all_streams if streamx['id'] in never_subscribed_stream_ids]
```

Comment:
Naming it stream was raising lint error(stream is already defined above at Line 2561).